ml-agents/mlagents/trainers/sac/optimizer_torch.py

```diff
@@ -257,8 +272,9 @@ def sac_value_loss(
                 )
                 value_losses.append(value_loss)
         else:
+            disc_log_probs = log_probs[:, self.policy.continuous_act_size :]
```

Comment:
We should be able to accomplish this with a `zip(entropy_coeffs, log probs)` of some sort